src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
```

Comment:
I think the spacing in your java files is a little inconsistent. We want to have the same indent everywhere. So if our indent is 2 spaces, we want to do it for all indents. I think you have a 1 space indent for you class methods/attributes. Please fix in both java files.